includes/template-functions.php

```diff
@@ -1080,9 +1080,12 @@ function the_no_permissions_message( $post_id ) {
  * Output the sensei excerpt
  *
  * @since 1.9.0
+ * @deprecated
```

Comment:
We should also add the version this was deprecated in. This will likely go in 3.2.0.